jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -170,10 +170,13 @@ const useFeedReducer = <State, Action>(reducer: (s: State, a: Action) => State,
   return [state, newDispatch]
 }
 
-const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = ({plugin, sortingObject, recordingObject, sortingInfo, recordingInfo, plugins, calculationPool, model, curationUri}) => {
+const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = ({plugin, sortingObject, recordingObject, plugins, calculationPool, model, curationUri}) => {
   const [sorting, setSorting] = useState<Sorting | null>(null)
   const [recording, setRecording] = useState<Recording | null>(null)
 
+  const sortingInfo = useSortingInfo(sorting?.sortingObject, sorting?.recordingObject)
```

Comment:
Introducing some very convenient custom hooks: useSortingInfo and useRecordingInfo. More about that further down.

Reply:
Yes that's good summary. Sometimes they are still passed in (as a remnant of previous method), but we may want to uniformize that at some point. On the other hand, maybe its good for low-level components to not refer to hither.